features/step_definitions/user_logout_steps.rb

```diff
@@ -0,0 +1,13 @@
+Given(/^I am on the laundry room page$/) do
+  visit '/laundry_room'
+end
+
+
+Given(/^I am logged in as that user$/) do
+  user = User.last
+  login_as(user)
+end
```

Comment:
Saving an object in an instance variable makes it possible to access it across step definitions.